.github/workflows/build-matrix.org.yml

```diff
@@ -0,0 +1,137 @@
+name: "🌐 Build the matrix.org website"
+on:
+  push:
+    branches: master
+  pull_request:
+  workflow_dispatch:
+jobs:
+  build-spec:
+    name: "📖 Spec"
+    runs-on: ubuntu-latest
+    container: "python:3.6"
+    steps:
+      - name: "📥 Source checkout"
+        uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - name: "📥 matrix-doc OpenAPI artifact download"
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: main.yml
+          workflow_conclusion: success
+          repo: ${{ github.repository_owner }}/matrix-doc
+          branch: ${{ github.ref }}
+          name: openapi-artifact
+      - name: "📖 Update docs"
+        run: |
+          python3 -m venv env
+          env/bin/pip install requests
+          . env/bin/activate; scripts/update_docs.sh
+      - name: "📦 Tarball creation"
+        run: tar -czf styled-docs.tar.gz content/docs
+      - name: "📤 Artifact upload"
+        uses: actions/upload-artifact@v2
+        with:
+          name: styled-docs-artifact
+          path: styled-docs.tar.gz
+  build-jekyll:
```

Comment:
please could you copy over the comments from the BK pipeline - https://github.com/matrix-org/matrix.org/blob/master/.buildkite/pipeline.yaml#L18 for example. Likewise the other comments in that file are worth keeping.

(indeed, there's a particular gotcha here in that the Dockerfile is in the `.buildkite` directory. We should move it eventually, but let's leave a comment for now)